specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,760 @@
+                        "$ref": "#/definitions/PublicKeysProperties"
+                    },
+                    "description": "This is information about the SSH certificate"
+                },
+                "platformFaultDomain": {
+                    "type": "string",
+                    "description": "This is the fault domain in which the VM."
+                },
+                "platformUpdateDomain": {
+                    "type": "string",
+                    "description": "This is the update domain in which the VM."
+                },
+                "provider": {
+                    "type": "string",
+                    "description": "This is the provider of the VM."
+                },
+                "publisher": {
+                    "type": "string",
+                    "description": "This is the publisher of the VM image."
+                },          
+                "resourceGroupName": {
+                    "type": "string",
+                    "description": "This is the resource group for the VM."
+                },
+                "sku": {
+                    "type": "string",
+                    "description": "This is the specific SKU for the VM image."
+                },          
+                "subscriptionId": {
+                    "type": "string",
+                    "description": "This is the Azure subscription for the VM."
+                },
+                "tags": {
+                    "type": "string",
+                    "description": "This is the list of tags for your VM."
+                },
+                "version": {
+                    "type": "string",
+                    "description": "This is the version of the VM image."
+                },
+                "vmId": {
+                    "type": "string",
+                    "description": "This is the unique identifier for the VM."
+                },
+                "vmScaleSetName": { 
+                    "type": "string",
+                    "description": "This is the resource name of the VMSS."
+                },
+                "vmSize": {
+                    "type": "string",
+                    "description": "This is the size of the VM."
+                },
+                "zone": {
+                    "type":"string",
+                    "description":"This is the availability zone of the VM."
+                }
+            }, 
+            "description": "Compute Metadata"
+        },
+        "Network": {
+            "type": "object",
+            "properties": {
+                "interface": {
+                    "type":"array",
+                    "items": {
+                        "type": "object",
+                        "$ref": "#/definitions/NetworkInterface"
+                    }
+                }            
+            },
+            "description": "Network Metadata"
+        },
+        "NetworkInterface": {
+            "type": "object",
```

Comment:
missing description. Please make sure every model, property, operation, parameter has correct and complete description. Spec description translates to api doc on docs.microsoft.com and sdk api docs as well. Missing descriptions end up providing a sub par developer experience.